tests/test_store.py

```diff
@@ -444,3 +466,65 @@ def test_upsertTable(self, helpers, synapse_store, config, projectId, datasetId)
         assert IDs.size == 8
         # delete table        
         synapse_store.syn.delete(tableId)
+
+
+class TestDownloadManifest:
+    @pytest.mark.parametrize("datasetFileView", [{"id": ["syn51203973", "syn51203943"], "name": ["synapse_storage_manifest.csv", "synapse_storage_manifest_censored.csv"]}, {"id": ["syn51203973"], "name": ["synapse_storage_manifest.csv"]}, {"id": ["syn51203943"], "name": ["synapse_storage_manifest_censored.csv"]}])
+    def test_get_manifest_id(self, synapse_store, datasetFileView):
+        # rows that contain the censored manifest
+        datasetFileViewDataFrame = pd.DataFrame(datasetFileView)
+        row_censored = datasetFileViewDataFrame.loc[datasetFileViewDataFrame['name'] == "synapse_storage_manifest_censored.csv"]
+        if len(row_censored) > 0:
```

Comment:
Im assuming if row_censored returns no censored manifests you will return columns but no rows, which is why you're looking for the length of the df? Can you do ` if not row_censored.empty` for a more compact solution?

Reply:
Yeah that works. Let me make that change. Thanks.